formatters/html/html.go

```diff
@@ -384,3 +386,25 @@ func splitTokensIntoLines(tokens []*chroma.Token) (out [][]*chroma.Token) {
 	}
 	return
 }
+
+// formatStyle removes empty style rules and
+// appends a semicolon to all style rules.
+//
+// Turns `{ ; vertical-align: top; border: 0 }`
+// into `{ vertical-align: top; border: 0; }`.
+func formatStyle(s string) string {
```

Comment:
This is a good thing to fix, but should be done at a lower level before the ; is added altogether.

Reply:
I've played with your lower level idea before, but opted for a final fail-safe in the end, which would ensure proper output of rules, even if css rule concatenation got of hand.
Because… well, bugs were introduced and few noticed, that the output was broken. So I liked this approach of ensuring, that the css stuff stayed correct.

I'll take a look at the lower level. Most issues seems to stem from the line number feature.